server/interceptor/interceptor_test.go

```diff
@@ -538,24 +577,27 @@ func TestTrillianInterceptor_QuotaInterception_ReturnsTokens(t *testing.T) {
 
 func TestTrillianInterceptor_NotIntercepted(t *testing.T) {
 	tests := []struct {
-		req interface{}
+		method string
+		req    interface{}
 	}{
 		// Admin
-		{req: &trillian.CreateTreeRequest{}},
-		{req: &trillian.ListTreesRequest{}},
+		{method: "/trillian.TrillianAdmin/CreateTree", req: &trillian.CreateTreeRequest{}},
```

Comment:
Consider extending the test coverage to catch bugs where the interceptor is accidentally universally disabled?

Reply:
Not sure I follow...
The existing tests verify that the handlers are running for the expected methods. 
If the interceptors get disabled these tests fail when they test for the presence of tree in `ctx`

Reply:
ok, then we're good :)